mavros_extras/src/plugins/vision_pose_estimate.cpp

```diff
@@ -111,18 +134,22 @@ class VisionPoseEstimatePlugin : public plugin::PluginBase,
     auto cov_ned = ftf::transform_frame_enu_ned(cov);
     ftf::EigenMapConstCovariance6d cov_map(cov_ned.data());
 
-    auto urt_view = Eigen::Matrix<double, 6, 6>(cov_map.triangularView<Eigen::Upper>());
-    ROS_DEBUG_STREAM_NAMED("vision_pose", "Vision: Covariance URT: " << std::endl << urt_view);
+    RCLCPP_INFO_STREAM(
+          get_logger(),
+          "Listen to vision transform " << tf_frame_id <<
+            " -> " << tf_child_frame_id);
+
 
     mavlink::common::msg::VISION_POSITION_ESTIMATE vp{};
 
-    vp.usec = stamp.toNSec() / 1000;
+    vp.usec = stamp.nanoseconds() / 1000;
     // [[[cog:
     // for f in "xyz":
     //     cog.outl("vp.%s = position.%s();" % (f, f))
     // for a, b in zip("xyz", ('roll', 'pitch', 'yaw')):
     //     cog.outl("vp.%s = rpy.%s();" % (b, a))
     // ]]]
+
```

Comment:
Please don't change cog-generated code.

Reply:
I think this was caused by my sanitize PR: https://github.com/mavlink/mavros/pull/1603
I didn't find a way to get rid of the warnings of the `\` in the end without creating that extra blank line. So all the blame on me :(